packages/graphql/src/translate/projection/elements/create-projection-subquery.ts

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Similarly to the other PR I believe this file could benefit from a little re-arranging.